playwright.config.ts

```diff
@@ -0,0 +1,52 @@
+import { defineConfig, devices } from '@playwright/test';
+import path from 'path';
+
+import { PluginOptions } from '@grafana/plugin-e2e';
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig<PluginOptions>({
+  testDir: './plugin-e2e',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
```

Comment:
Why do we want to opt out of parallel tests on CI? Is this unsupported?

Reply:
Following Playwright [standards](https://playwright.dev/docs/ci#workers) here. We may be able to fine-tune this in the future when we have more tests, but let's start simple. The tests only takes a few seconds to run anyway.